components/match2/wikis/halo/match_summary.lua

```diff
@@ -144,14 +144,35 @@ function CustomMatchSummary._createMapRow(game)
 		centerNode:addClass('brkts-popup-spaced-map-skip')
 	end
 
+	local getScoreValue = function(opponentIndex)
```

Comment:
if we add the flip option maybe the following might be an option too (code wise)
```lua
	local getScoreValue = function(opponentIndex)
		local score = game.scores[opponentIndex]
		if not score then return end
		local points = game.extradata['points' .. opponentIndex]
		local displayParts = Array.extend(
			score,
			points and ('(' .. points .. ')') or nil,
		)
		if opponentIndex == 2 then
			displayParts = Array.reverse(displayParts)
		end
		return table.concat(displayParts, ' ')
	end
```

(still on phone)

Reply:
Don't think this is really necessary as there won't be any further parts, but complicates the function a lot